relay-genesis/relay-genesis.js

```diff
@@ -0,0 +1,42 @@
+const BitcoinRpc = require('bitcoind-rpc')
+const Bluebird = require('bluebird')
+const config = require("../configs/relay-genesis/config.json")
+
+const bitcoinRpc = new BitcoinRpc(config)
+Bluebird.promisifyAll(bitcoinRpc)
+
+async function run() {
+    try {
+        const blockCount = (await bitcoinRpc.getblockcountAsync()).result
+
+        const genesisBlock = blockCount - 5
+        const genesisBlockHash = (await bitcoinRpc.getblockhashAsync(genesisBlock)).result
+        const genesisBlockHeader = (await bitcoinRpc.getblockheaderAsync(genesisBlockHash, false)).result
+
+        const epochStartBlock = genesisBlock - (genesisBlock % 2016)
+        const epochStartBlockHash = (await bitcoinRpc.getblockhashAsync(epochStartBlock)).result
+        const epochStartBlockHashReversed = changeEndianness(epochStartBlockHash)
+
+        const result = {
+            genesis: `0x${genesisBlockHeader}`,
+            height: genesisBlock,
+            epochStart: `0x${epochStartBlockHashReversed}`
+        }
+
+        console.log(JSON.stringify(result))
+    } catch (e) {
+        console.log(e)
+    }
+}
+
+const changeEndianness = (string) => {
+    const result = [];
+    let len = string.length - 2;
```

Comment:
Are we doing `- 2` because of `0x` prefix? Maybe we want to be more generic and support cases with and without `0x` prefix?

Reply:
Nope, it's not about the prefix. One byte corresponds to two characters and we are starting from the end. So, if you want to take the last byte and push it to the `result` array you have to do `string.substr(string.length-2, 2)` first.